algebird-test/src/test/scala/com/twitter/algebird/BatchedTest.scala

```diff
@@ -52,6 +52,30 @@ class BatchedLaws extends CheckProperties {
   testBatchedMonoid[String]("String", 10)
   testBatchedMonoid[String]("String", 100)
   testBatchedMonoid[String]("String", 1000000)
+
+  def testBatchedSemigroup[A: Arbitrary: Semigroup](name: String, size: Int): Unit = {
+    implicit val m: Semigroup[Batched[A]] = Batched.compactingSemigroup[A](size)
+    property(s"CountMinSketch[$name] batched at $size is a Semigroup") {
```

Comment:
what is countminsketch got to do with anything here?

Reply:
copy-pasta... I didn't read at all but it comes directly from `def testBatchedMonoid`